AppofML_ITP449/Code_repo/sandboxes/sandbox14-svm_irisset.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sb
+
+pd.set_option('display.max_columns', None)
+iris_df = pd.read_csv(
+    '/Users/shantanujhaveri/Desktop/work-git/GitHub/Learning-Modules_introML/AppofML_ITP449/Code_repo/sandboxes/sandbox-data/iris.csv')
+# print(iris_df.info())
+# print(iris_df.Species.unique())
+iris_df_sepal = iris_df.drop(columns=['Petal length', 'Petal width'])
+plt.figure(1)
+sb.scatterplot(data=iris_df_sepal, x='Sepal width', y='Sepal length', hue='Species') # Professor made this scatterplot
+# in the most gross way possible, if time allots I will come back and add that
+plt.show()
+plt.figure(2)
+iris_df_petal = iris_df.drop(columns=['Sepal length', 'Sepal width'])
+sb.scatterplot(data=iris_df_petal, x='Petal width', y='Petal length', hue='Species')
```

Comment:
this is so spicy. prof reza really coded this simple graph in like 30 lines of code... smh